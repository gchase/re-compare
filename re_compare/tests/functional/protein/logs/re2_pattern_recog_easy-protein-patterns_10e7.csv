regex,matchtype,matchnumber,time(ms),span
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,first_match,0,93.4288501739502,"[-1, -1]"
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,all_matches,0,93.4288501739502,"[-1, -1]"
DHKNLD.,first_match,1,0.6611347198486328,"[613257, 613263]"
DHKNLD.,consecutive,1,0.6611347198486328,"[613257, 613263]"
DHKNLD.,consecutive,2,0.0059604644775390625,"[613397, 613403]"
DHKNLD.,all_matches,2,0.6670951843261719,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,first_match,1,0.5748271942138672,"[98919, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,1,0.5748271942138672,"[98919, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,2,0.010013580322265625,"[99268, 99273]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,all_matches,2,0.5848407745361328,"[-1, -1]"
(GI)(G|R)P.Y..K..,first_match,1,0.47516822814941406,"[214952, 214962]"
(GI)(G|R)P.Y..K..,consecutive,1,0.47516822814941406,"[214952, 214962]"
(GI)(G|R)P.Y..K..,consecutive,2,0.010013580322265625,"[215383, 215393]"
(GI)(G|R)P.Y..K..,all_matches,2,0.4851818084716797,"[-1, -1]"
G|S|T|A|D|N|E|K|R,first_match,1,0.08702278137207031,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,1,0.08702278137207031,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,2,0.0069141387939453125,"[4, 4]"
G|S|T|A|D|N|E|K|R,all_matches,2,0.09393692016601562,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),first_match,1,0.5109310150146484,"[49973, 49978]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,1,0.5109310150146484,"[49973, 49978]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,2,0.014066696166992188,"[50386, 50391]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),all_matches,2,0.5249977111816406,"[-1, -1]"
(N|S.TNH.Y)(F|W),first_match,1,0.1361370086669922,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,1,0.1361370086669922,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,2,0.013828277587890625,"[441, 442]"
(N|S.TNH.Y)(F|W),all_matches,2,0.1499652862548828,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),first_match,1,0.29778480529785156,"[11709, 11713]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,1,0.29778480529785156,"[11709, 11713]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,2,0.21696090698242188,"[58929, 58933]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),all_matches,2,0.5147457122802734,"[-1, -1]"
P..L...KWE.,first_match,1,4.71806526184082,"[4211985, 4211995]"
P..L...KWE.,consecutive,1,4.71806526184082,"[4211985, 4211995]"
P..L...KWE.,consecutive,2,0.009059906005859375,"[4212434, 4212444]"
P..L...KWE.,all_matches,2,4.72712516784668,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,first_match,0,40.96698760986328,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,all_matches,0,40.96698760986328,"[-1, -1]"
