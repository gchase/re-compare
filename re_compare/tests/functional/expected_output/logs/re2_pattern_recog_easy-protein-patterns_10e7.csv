regex,matchtype,matchnumber,time(ms),span
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,first_match,0,37.28008270263672,"[-1, -1]"
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,all_matches,0,37.28008270263672,"[-1, -1]"
DHKNLD.,first_match,1,0.7407665252685547,"[613257, 613263]"
DHKNLD.,consecutive,1,0.7407665252685547,"[613257, 613263]"
DHKNLD.,consecutive,2,0.0057220458984375,"[613397, 613403]"
DHKNLD.,all_matches,2,0.7464885711669922,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,first_match,1,0.5738735198974609,"[98919, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,1,0.5738735198974609,"[98919, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,2,0.009059906005859375,"[99268, 99273]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,all_matches,2,0.5829334259033203,"[-1, -1]"
(GI)(G|R)P.Y..K..,first_match,1,0.5331039428710938,"[214952, 214962]"
(GI)(G|R)P.Y..K..,consecutive,1,0.5331039428710938,"[214952, 214962]"
(GI)(G|R)P.Y..K..,consecutive,2,0.009059906005859375,"[215383, 215393]"
(GI)(G|R)P.Y..K..,all_matches,2,0.5421638488769531,"[-1, -1]"
G|S|T|A|D|N|E|K|R,first_match,1,0.09298324584960938,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,1,0.09298324584960938,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,2,0.0069141387939453125,"[4, 4]"
G|S|T|A|D|N|E|K|R,all_matches,2,0.09989738464355469,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),first_match,1,0.5099773406982422,"[49973, 49978]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,1,0.5099773406982422,"[49973, 49978]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,2,0.013113021850585938,"[50386, 50391]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),all_matches,2,0.5230903625488281,"[-1, -1]"
(N|S.TNH.Y)(F|W),first_match,1,0.1418590545654297,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,1,0.1418590545654297,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,2,0.014066696166992188,"[441, 442]"
(N|S.TNH.Y)(F|W),all_matches,2,0.15592575073242188,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),first_match,1,1.3241767883300781,"[11709, 11713]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,1,1.3241767883300781,"[11709, 11713]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,2,0.5118846893310547,"[58929, 58933]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),all_matches,2,1.8360614776611328,"[-1, -1]"
P..L...KWE.,first_match,1,7.222175598144531,"[4211985, 4211995]"
P..L...KWE.,consecutive,1,7.222175598144531,"[4211985, 4211995]"
P..L...KWE.,consecutive,2,0.007867813110351562,"[4212434, 4212444]"
P..L...KWE.,all_matches,2,7.230043411254883,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,first_match,0,47.63507843017578,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,all_matches,0,47.63507843017578,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),first_match,1,0.21910667419433594,"[830, 838]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),consecutive,1,0.21910667419433594,"[830, 838]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),consecutive,2,0.015974044799804688,"[996, 1004]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),all_matches,2,0.23508071899414062,"[-1, -1]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),first_match,1,0.6527900695800781,"[41351, 41357]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),consecutive,1,0.6527900695800781,"[41351, 41357]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),consecutive,2,0.3917217254638672,"[130717, 130723]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),all_matches,2,1.0445117950439453,"[-1, -1]"
((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A),first_match,0,10.557889938354492,"[-1, -1]"
((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A),all_matches,0,10.557889938354492,"[-1, -1]"
A|G....G,first_match,1,0.14209747314453125,"[15, 15]"
A|G....G,consecutive,1,0.14209747314453125,"[15, 15]"
A|G....G,consecutive,2,0.0059604644775390625,"[19, 19]"
A|G....G,all_matches,2,0.1480579376220703,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)G,first_match,1,0.32210350036621094,"[57782, 57786]"
((CP|.)(S|T)N)(I|L|V)G,consecutive,1,0.32210350036621094,"[57782, 57786]"
((CP|.)(S|T)N)(I|L|V)G,consecutive,2,0.010013580322265625,"[58001, 58005]"
((CP|.)(S|T)N)(I|L|V)G,all_matches,2,0.33211708068847656,"[-1, -1]"
